lib/rack/utf8_sanitizer.rb

```diff
@@ -62,7 +62,13 @@ def sanitize_rack_input(env)
       content_type &&= content_type.downcase
       return unless SANITIZABLE_CONTENT_TYPES.any? {|type| content_type == type }
       uri_encoded = URI_ENCODED_CONTENT_TYPES.any? {|type| content_type == type}
-      env['rack.input'] &&= sanitize_io(env['rack.input'], uri_encoded)
+
+      if env["rack.input"]
+        sanitized_input = sanitize_io(env['rack.input'], uri_encoded)
+
+        env['rack.input'] = sanitized_input
+        env['CONTENT_LENGTH'] &&= sanitized_input.size.to_s
```

Comment:
Tricky. I started to comment here that this should be `bytesize` but I see below in `#size` and in the tests that it already is. :100: